packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -145,10 +142,16 @@ async fn get_entity<P: StorePool + Send>(
 )]
```

Comment:
(comment for utoipa annotations)
We'd need to add something like `request_body = Option<Expression>` as an attribute to capture the body in the requests. This would go for all places where we have expression as a body. It isn't really a blocker right now, but we'll be adding it when we get to use queries from the typescript code. Just flagging, no need to apply this change :)

This of course also implies we might need to switch away from GETs in the read operations